src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/Maintainability/ReviewUnusedParametersTests.cs

```diff
@@ -1197,6 +1197,17 @@ public async Task CA1801_TopLevelStatements_NoDiagnostic()
             }.RunAsync();
         }
 
+        [Fact, WorkItem(4462, "https://github.com/dotnet/roslyn-analyzers/issues/4462")]
+        public async Task CA1801_CSharp_ImplicitRecord()
+        {
+            await new VerifyCS.Test
+            {
+                LanguageVersion = CodeAnalysis.CSharp.LanguageVersion.CSharp9,
+                ReferenceAssemblies = ReferenceAssemblies.Net.Net50,
+                TestCode = @"public record Person(string Name, int Age = 0);",
```

Comment:
You may want to add another test where the record has braces instead of a semi colon.

```csharp
"public record Person(string Name, int Age = 0) {}"
```

Reply:
What about:

```csharp
public record Person(string Name, int Age = 0)
{
    public int Age { get; init; }
}
```

AFAIL, the parameter `Age` is unused for this case.

Reply:
For the first example, do we agree that issue should be reported for the 2 arguments? Or is it the body of the record and not the record ctor?

Reply:
> For the first example, do we agree that issue should be reported for the 2 arguments? Or is it the body of the record and not the record ctor?

The first example I mentioned (`"public record Person(string Name, int Age = 0) {}"` is identical to `"public record Person(string Name, int Age = 0);"`).

Both compile to the same code. I think the test will just pass without any further modifications, but just to confirm.